doc/release_notes.rst

```diff
@@ -31,6 +31,8 @@ Upcoming Release
 
 * To enable better backwards compatibility with the `lopf` function, the ``Network.optimize`` functions has now the explicit keyword argument ``solver_options``. It takes a dictionary of options passed to the solver. Before, these were passed as keyword arguments to the ``Network.optimize`` function. Note that both functionalities are supported.
 
+* Fix an issue appeared when processing networks which were reduced to a set of isolated nodes in course of clustering.
```

Comment:
Could you make this release note a little more descriptive about what errors were encountered and how you fixed it?

Reply:
Tried to improve. Could you please check if now it is clear enough and not too technical?